ie-solver/boundaries/annulus.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019 John Paul Ryan
+#include <cmath>
+#include <iostream>
+#include "ie-solver/boundaries/annulus.h"
+#include "ie-solver/log.h"
+
+namespace ie_solver {
+
+void Annulus::initialize(int N, BoundaryCondition bc) {
+  N = 6 * (N / 6);
+
+
+  if (bc == BoundaryCondition::STOKES) {
+    boundary_values = ie_Mat(2 * N, 1);
+  } else {
+    boundary_values = ie_Mat(N, 1);
+  }
+  boundary_condition = bc;
+  points.clear();
+  normals.clear();
+  weights.clear();
+  curvatures.clear();
+
+
+  for (int i = 0; i < 5 * (N / 6); i++) {
+    double ang = i * 2.0 * M_PI / (5 * (N / 6));
+    double x = 0.5 + 0.25 * cos(ang);
```

Comment:
consider parametrizing a curve for the center of the inner circle to travel along using perturbation arg (see rswb boundary)